.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+# Java Gradle CircleCI 2.0 configuration file
```

Comment:
Revert this. We shouldn't this update on this PR

Reply:
I see, I don't have that file locally, it appears only on git hub so I assume circleci updated the PR when we installed it, that's why I told you tests are failing...